packages/core/src/dialects/postgres/connection-manager.ts

```diff
@@ -193,9 +193,9 @@ export class PostgresConnectionManager extends AbstractConnectionManager<PgConne
     });
 
     // Don't let a Postgres restart (or error) to take down the whole app
-    connection.once('error', error => {
+    connection.on('error', error => {
       connection._invalid = true;
-      debug(`connection error ${error.code || error.message}`);
+      debug(`connection error ${(error as unknown as { code: string }).code || error.message}`);
```

Comment:
Is this still needed here?

Reply:
Update; it is. When using once it sees `error` as `any`, but using `on` it is typed as an `Error` which does not have a code.

```suggestion
      debug(`connection error ${error.code || error.message}`);
```